src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -928,12 +928,12 @@ namespace Sapphire::Network::Packets::Server
 
     uint8_t unknown5;
     uint32_t unknown8;
-    uint16_t festivalId;
-    uint16_t additionalFestivalId;
     uint32_t unknown9;
     uint32_t unknown10;
+    uint32_t festivalId;
     uint32_t unknown11;
-    uint32_t unknown12[4];
+    uint32_t unknown12[3];
+    uint32_t additionalFestivalId;
```

Comment:
This is an InitZone packet from the Gold Saucer event:
`10 00 90 00 00 00 00 00 00 00 00 00 17 46 56 00`
`02 59 00 00 AA D8 CF 00 02 00 00 00 00 80 09 41`
`00 00 80 3F` **73 00 00 00** `00 00 00 00 00 00 00 00`
`00 00 00 00` **73 00 00 00** `00 00 00 00 00 00 00 00`
`00 00 00 00 59 1D 0D C2 F9 D5 23 3F B7 FF C7 42`
`00 01 01 04 04 01 04 03 01 04 02 00 00 00 00 00`

This is an InitZone packet from the Moonfaire event:
`17 00 89 00 00 00 00 00 00 00 00 00 C0 1E 81 00`
`02 39 10 00 AA E0 24 00 15 00 00 00 00 80 09 41`
`00 00 80 3F` **74 00 00 00** `00 00 00 00 00 00 00 00`
`00 00 00 00 `**74 00 00 00** `00 00 00 00 00 00 00 00`
`00 00 00 00 7B 94 75 43 2B DA 97 42 93 85 4D 44`
`00 01 04 04 01 01 01 04 01 02 01 00 00 00 00 00`

Because of these two festival InitZone packets, I think they have changed something in the festival system.